packages/fiber/tests/web/canvas.test.tsx

```diff
@@ -39,6 +39,21 @@ describe('web Canvas', () => {
     expect(ref.current).toBeDefined()
   })
 
+  it('should forward ref three object', async () => {
```

Comment:
To clear up a few questions: refs all have the same signature, it's that the useRef's behavior is different under the hood.

That TS issue might be because your system isn't creating symlinks -- something to look into.